dabuild.in

```diff
@@ -73,6 +73,26 @@ ABUILD_VOLUMES="-v ${HOME}/.abuild:/home/builder/.abuild \
   -v ${PWD%/aports/*}/aports:/home/builder/aports \
   -v ${ABUILD_PACKAGES}:/home/builder/packages"
 
+## if we have ~/.ccache on host, bind-mount it; else use named per-arch volumes
+if [ -w "$HOME/.ccache" ]; then
+  ccvol=$HOME/.ccache/$DABUILD_ARCH
+  if [ ! \( -d "$ccvol" -a -w "$ccvol" \) ]; then
+    mkdir $ccvol
+  fi
+else
+  ccvol=abuild-$ABUILD_VERSION-$DABUILD_ARCH-ccache
```

Comment:
ABUILD_VERSION is a kind of confusing. Can we globally rename this to ALPINE_BRANCH or ALPINE_RELEASE of which the later sounds preferable?

Reply:
:+1:

Reply:
Hi, not sure you noticed but i switched this repo to gitlab.
Please dont push to git.a.o anymore.